cg/meta/workflow/microsalt.py

```diff
@@ -248,3 +251,23 @@ def request_removed_flowcells(self, ticket):
             elif flowcell_obj.status != "ondisk":
                 LOG.warning(f"{flowcell_obj.name}: {flowcell_obj.status}")
         self.db.commit()
+
+    def get_deliverables_to_store(self) -> list:
+        """Retrieve a list of microbial deliverables files for orders where analysis finished
+        successfully, and are ready to be stored in Housekeeper"""
+        deliverables_to_store = []
+        for case_object in self.db.cases_to_store(pipeline="microbial"):
+            deliverables_file = self.get_deliverables_file_path(order_id=case_object.name)
+            if Path(deliverables_file).exists():
+                deliverables_to_store.append(deliverables_file)
+        return deliverables_to_store
+
+    def get_deliverables_file_path(self, order_id: str) -> str:
```

Comment:
Why not return a Path object and convert to whatever you like when needed?

Reply:
tbh I straight up stole this from somewhere else :pirate_flag:

I changed it